### PR TITLE
README and misc updates

### PR DESCRIPTION
**Pull request by MikeJeffers**

### CODE REVIEW

**Review by clouths — DISMISSED** [review]
I approve.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Your code will be reviewed and merged if it does the following.
+Either:
+ - Addresses or implements an OP-Project identified bug, feature, or change.
+ - Enhances, optimizes, or extends existing functionality.
+ - Refactors that improves maintainability  of the codebase.
+
+Your code must:
+ - Not break style conventions within the codebase.
+ - Be either ungrokable, insufficiently documented (self-/comments) or otherwise unmaintainable by other experienced developers.
```
Comment (nref-dan) [inline CONTRIBUTING.md]: Ummm, I think you mean the opposite :D

Comment (nref-dan) [reply CONTRIBUTING.md]: Also, Grok, really? 😛

**Review by nref-dan — CHANGES_REQUESTED** [review]
Line 17 is problematic and needs changing. Also the README is detected as a binary file. It might be worth deleting your local and typing it out on Notepad in ASCII mode. Possible hidden Unicode character, as I'm unsure why else it would think that.

Final point, I think we should include some basic build/startup instructions in the README. At the very least, a list of tools/frameworks needed to build/F5 the project.

**Comment by MikeJeffers** [comment]
> 
> 
> Line 17 is problematic and needs changing. Also the README is detected as a binary file. It might be worth deleting your local and typing it out on Notepad in ASCII mode. Possible hidden Unicode character, as I'm unsure why else it would think that.
> 
> Final point, I think we should include some basic build/startup instructions in the README. At the very least, a list of tools/frameworks needed to build/F5 the project.

Totally agree, please add and improve this!  I am just trying to kick-start some documentation and clarifying text to get people oriented at least.

**Comment by MikeJeffers** [comment]
@Ketsuekiame need re-review before we can merge

**Review by clouths — DISMISSED** [review]
I approve this update!
Also the content on the website seems excellent so far, other than maybe adding the links on the navigation page, everything is fine!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Reviewers will point out when these tenets are not followed, but it will save everyone time if you adhere to these rules the first time.  
+
+Special Notes:
+The PerpetuumServer is highly coupled in some instances to DB state.  The OPDB has its own patching source control model which you should learn and follow.  If your pull request also requires database changes:
+ - Submit your PR to the server, and another PR of your SQL patch to the OPDB development branch
+ - **Include a comment in the PR note that the server and DB patch are interdependent**
+
+## OP-project - Cross Repo issue and feature planning
+As mentioned above, all action items are collected in one kanban board for Server related development.  These issues should be pointed to with all PR's made.  A PR should address at most one OP-Project issue.  When a server change requires a DB change, use the issue too as a mechanism to indicate the coupling.
```
Comment (nref-dan) [inline CONTRIBUTING.md:29]: `issue too`? Unsure what you are trying to say here so I'm not sure what to suggest.

**Review by nref-dan — CHANGES_REQUESTED** [review]
I'm also going to need to change the readme file manually because github sees it as a binary file and won't let me review it.

**Review by nref-dan — APPROVED** [review]

**Pull request merged**